src/main/java/edu/harvard/iq/dataverse/branding/BrandingUtil.java

```diff
@@ -1,22 +1,37 @@
 package edu.harvard.iq.dataverse.branding;
 
+import edu.harvard.iq.dataverse.DataverseServiceBean;
 import edu.harvard.iq.dataverse.util.BundleUtil;
 import java.util.Arrays;
+
+import javax.ejb.EJB;
+import javax.ejb.Singleton;
 import javax.mail.internet.InternetAddress;
 
+@Singleton
 public class BrandingUtil {
 
-    public static String getInstallationBrandName(String rootDataverseName) {
-        return rootDataverseName;
+    @EJB
+    static DataverseServiceBean dataverseService;
```

Comment:
I didn't even know this is possible. It doesn't feel very beanish, to be honest.

Reply:
reverted to using a helper.